tests/stress_tests/.automation/memcheck_tests/nightly_configs/desktop_references_config.xml

```diff
@@ -432,7 +432,7 @@
         <model path="public/mtcnn/mtcnn-r/FP16/mtcnn-r.xml" precision="FP16" test="infer_request_inference" device="GPU" vmsize="1954758" vmpeak="2039954" vmrss="286015" vmhwm="382257" />
         <model path="public/mtcnn/mtcnn-r/FP16/mtcnn-r.xml" precision="FP16" test="inference_with_streams" device="CPU" vmsize="1001962" vmpeak="1001962" vmrss="36218" vmhwm="36218" /> # Values from {"commit_id": "403339f8f470c90dee6f6d94ed58644b2787f66b", "commit_date": "2022-01-19 14:13"} and *= 1.3
         <model path="public/mtcnn/mtcnn-r/FP16/mtcnn-r.xml" precision="FP16" test="inference_with_streams" device="GPU" vmsize="2052585" vmpeak="2052585" vmrss="287752" vmhwm="383385" />
-        <model path="public/mtcnn/mtcnn-r/FP32/mtcnn-r.xml" precision="FP32" test="create_exenetwork" device="CPU" vmsize="892860" vmpeak="973684" vmrss="26197" vmhwm="26197" />
+        <model path="public/mtcnn/mtcnn-r/FP32/mtcnn-r.xml" precision="FP32" test="create_exenetwork" device="CPU" vmsize="901846" vmpeak="987043" vmrss="34580" vmhwm="34580" /> # Values from {"commit_id": "05684b7cd0cef308de3a81fc3bc1ab4273998f58", "commit_date": "2022-01-27 14:41"} and *= 1.3
```

Comment:
@just-sparta @dkrupnov do you have a confirmation such huge growth is expected?

Reply:
@dmitry-gorokhov Please review changes (additional info in Teams)